R/labels.R

```diff
@@ -1,44 +1,71 @@
 # These are S3 methods for the labelled objects coming from {labelVector}
-# The {vctrs} package needs these for working in dplyr 
+# The {vctrs} package needs these for working in {dplyr} 
 
 #' @importFrom vctrs vec_cast
 #' @importFrom vctrs vec_ptype2
 
-# both labelled
+#### Checking types of objects ####
+
+##### both labelled #####
 #' @export
 vec_ptype2.labelled.labelled <- function(x, y, ...) {
   x
 }
 
-# char
+##### char #####
+#' @export
+vec_ptype2.labelled.character <- function(x, y, ...) {
+  x
+}
+
 #' @export
 vec_ptype2.character.labelled <- function(x, y, ...) {
   y
 }
 
-# integer
+##### integer ##### 
+#' @export
+vec_ptype2.labelled.integer <- function(x, y, ...) {
+  x
+}
+
 #' @export
 vec_ptype2.integer.labelled <- function(x, y, ...) {
   y
 }
 
-# double
+##### double ##### 
+#' @export
+vec_ptype2.labelled.double <- function(x, y, ...) {
+  x
+}
+
 #' @export
 vec_ptype2.double.labelled <- function(x, y, ...) {
   y
 }
 
-# logical
+
+##### logical #####
+#' @export
+vec_ptype2.labelled.logical <- function(x, y, ...) {
+  x
+}
+
 #' @export
 vec_ptype2.logical.labelled <- function(x, y, ...) {
   y
 }
 
+#### Casting to fix attributes ####
+# These functions all return the object `x` itself if it was already labelled 
+#   or `x` with the label from the `to` object.
 
-# both labelled
+##### both labelled #####
```

Comment:
these stand out much better now with the quad hashes

Reply:
This information should be added to the roxygen documentation using either the `@details` tag